crypto/Pos/Crypto/Hashing.hs

```diff
@@ -143,7 +144,7 @@ abstractHash = unsafeAbstractHash
 unsafeAbstractHash
     :: (HashAlgorithm algo, Bi a)
     => a -> AbstractHash algo b
-unsafeAbstractHash = AbstractHash . Hash.hash . Bi.serialize'
+unsafeAbstractHash = AbstractHash . Hash.hashlazy . Bi.serializeWith 32 1024
```

Comment:
This is `serialize`, but with different numbers. Why? Can you please leave the comment about it in the code?

Reply:
Will do. For the record: `32` is chosen because the hash we actually use is 32 bytes. Really it ought to take the `hashDigestSize (undefined :: algo)`. I'll do that instead.

Reply:
Oh nevermind, this looks like a mistake.

Reply:
Serializing a digest should use this, but here we're serializing the thing to be hashed.